py/examples/table_search_groupable.py

```diff
@@ -0,0 +1,102 @@
+import random
```

Comment:
This example made the review buttery smooth, but we have plenty of specialized table examples so will not merge this one. Please remove it.

Reply:
done